workload-extensions/oci-lz-ext-ocvs/op02-ocvs-workload-extension/oci-credentials.tfvars.json.template

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
@rphibbert @jpalmeiro should we maybe have a shared folder at top lavel for shared content e.g explaining authentication and then just link it from all the guides instead of having a copy everywhere

Reply:
makes sense, add it to your backlog